pom.xml

```diff
@@ -12,14 +12,21 @@
     <description>Java SDK for Dapr Pluggable Components.</description>
     <url>https://dapr.io</url>
 
+    <modules>
+        <module>components-java-sdk-autogen</module>
+    </modules>
+
     <properties>
+        <dapr.proto.baseurl>https://raw.githubusercontent.com/dapr/dapr/v1.9.5/dapr/proto</dapr.proto.baseurl>
```

Comment:
neat!

Reply:
I wish I could take credit for this but this comes right from `dapr/dapr`.